cmd/fyne/internal/commands/get.go

```diff
@@ -81,7 +88,7 @@ func (g *Getter) Get(pkg string) error {
 	if repo.VCS.Name != "Git" {
 		return errors.New("unsupported VCS: " + repo.VCS.Name)
 	}
-	cmd := exec.Command("git", "clone", repo.Repo, "--depth=1", path)
+	cmd := execabs.Command("git", "clone", repo.Repo, "--depth=1", path)
```

Comment:
The execabs is not needed any more I think, because Go 1.19 fixed the underlying issues.